content/english/anti-vegan/no-food-is-fully-vegan.md

```diff
@@ -11,33 +11,103 @@ weight: 5
 
-Ed Winters (aka Earthling Ed) hits the nail on the head with zir comparison in zir book '30 Non Vegan Excuses'.
+* A cattle farmer knows that the cattle will be slaughtered and intends for them to be slaughtered.
 
-> If someone was driving their car and they accidentally hit a dog, that would not be the same as if they purposefully drove after the dog until they ran them over.  
-> - Ed Winters - 30 Non Vegan Excuses - p72
+* A crop farmer knows that some animals (e.g rabbits, birds, rodents) may be killed during harvest but the intention is only to harvest crops. (The crop farmer would far prefer the animals were not in the field).
+ 
+Consider the following thought experiment:
 
-Intention matters in the morality of your actions. The unintentional deaths of earthworms when tilling, or field mice when harvesting are not _necessary_ for the production of crops, it is not the intention to kill animals when harvesting crops. It is the definitionally the intention when raising animals for slaughter.
+> You are controlling a tram. As long as you hold a lever the tram will continue on its current track.  However, you know there are numerous people on the track ahead who will be killed if the tram continues.  
+> If you release the lever the tram will change tracks. You suspect there are *very probably* people on the other track as well.
 
-This distinction falls down somewhat when we consider pesticides, animal-based fertilizers, and beekeeping for pollination. This is why some vegans seek out foods from [vegan farms](https://www.viva.org.uk/what-we-do/vegan-farming). Ultimately however each person's 'possible and practicable' is a line they have to draw for themselves and this may not be within the means or motivations of every vegan. It is not necessary to be perfect to be vegan.
+1. Continuing to hold the lever is to kill with both foreknowledge and intent.
 
-### Moral Consistency
+2. Releasing the lever is to have the foreknowledge that some people might be killed.
 
-Trying to trip vegans up by exposing real or imagined inconsistencies is a common avenue of argument against veganism. The reality is that vegans (being human) are typically not perfect. Vegans are subject to the constraints of time, money, knowledge, and availability of resources amongst a million other factors. We quoted the definition of veganism in our quick response but it's so important it bears repeating: 
+Knowingly risking some deaths is morally preferable to knowingly killing with intent.
 
-> Veganism is a way of living which seeks to exclude, as far as is possible and practicable, all forms of exploitation of, and cruelty to, animals  for food, clothing or any other purpose.  
+Humans hold a lever which results in the intentional deaths of millions of animals. It's time to let go of the lever; the foreknowledge that some animals will probably die as a result of crop farming is not an argument against veganism.
```

Comment:
I love this so much, if we ever sell t-shirts we need one that says "let go of the lever".